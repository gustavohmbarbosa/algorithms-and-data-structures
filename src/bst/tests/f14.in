1
50
1
70
1
30
1
20
1
40
1
80
14
70
14
50
99