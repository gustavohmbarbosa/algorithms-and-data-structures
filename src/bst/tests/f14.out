[80]
[20][30][40][70][80]
